oauth2_http/javatests/com/google/auth/oauth2/AwsCredentialsTest.java

```diff
@@ -73,6 +82,9 @@
         }
       };
 
+  private static final Map<String, Object> emptyMetadataHeaders = Collections.emptyMap();
```

Comment:
Java style guide: Constant names use UPPER_SNAKE_CASE: all uppercase letters, with each word separated from the next by a single underscore.

May want to inline. Up to you.